src/board.cpp

```diff
@@ -64,11 +64,13 @@ void setupHW()
 
 void setEPaperPowerOn(bool on)
 {
-  // use HIGH/LOW notation for better readability
-#if (defined ES3ink) || (defined MakerBadge_revD) || (defined SVERIO_PAPERBOARD_SPI)
+// use HIGH/LOW notation for better readability
```

Comment:
This whole section shouldn't be touched

Reply:
No, because it's part of this commit, so it will create a mess. Will you clean it up, or should I do it?